db/migrate/20150504075221_populate_participating.rb

```diff
@@ -0,0 +1,16 @@
+class PopulateParticipating < ActiveRecord::Migration
+  def up
+    progress_bar = ProgressBar.create( format: "(\e[32m%c/%C\e[0m) %a |%B| \e[31m%e\e[0m ",
+                                       progress_mark: "\e[32m/\e[0m",
+                                       total: Discussion.count )
+    Discussion.reset_column_information
```

Comment:
ok. We'll need to pull the data migration out of the schema migration. Can you throw this into the DiscussionService or something please?